README.md

```diff
@@ -77,25 +76,27 @@ platform. Packages included, by default, in Tanzu Community Edition are:
 | ExternalDNS | Provides discoverability of services via public DNS | [ExternalDNS package docs](./addons/packages/external-dns) |
-| Harbor | Provides cloud native container registry service | [Harbor package docs](./addons/packages/harbor) |
 | Fluent-Bit | Log processor and forwarder | [Fluent Bit package docs](./addons/packages/fluent-bit) |
 | Gatekeeper | Provides policy enforcement within clusters | [Gatekeeper package docs](./addons/packages/gatekeeper) |
 | Grafana | Metrics visualization and analytics | [Grafana package docs](./addons/packages/grafana) |
-| Knative Serving | Provides serving functionality to clusters | [knative serving package docs](./addons/packages/knative-serving) |
+| Harbor | Provides cloud native container registry service | [Harbor package docs](./addons/packages/harbor) |
+| Knative Serving | Provides serving functionality to clusters | [Knative serving package docs](./addons/packages/knative-serving) |
+| Load Balancer Operator | Provides load balancer integrations to clusters | [Load Balancer Operator package docs](./addons/packages/ako-operator) |
+| Local Path Storage| Provides local path storage | [Local path storage docs](./addons/packages/local-path-storage) |
+| Kpack | Utilizes unprivileged Kubernetes primitives to provide builds of OCI images | [Kpack docs](./addons/packages/kpack) |
+| Multus CNI | Provides ability for attaching multiple network interfaces to pods in Kubernetes | [Multus CNI package docs](./addons/packages/multus-cni) |
 | Prometheus | Time series database for metrics. Includes AlertManager | [Prometheus package docs](./addons/packages/prometheus) |
+| Sriov Network Device Plugin | The SR-IOV Network Device Plugin is Kubernetes device plugin for discovering and advertising SR-IOV virtual functions (VFs) available on a Kubernetes host. | [Sriov Network Device Plugin package docs](./addons/packages/sriov-network-device-plugin) |
 | Velero | Provides disaster recovery capabilities | [Velero package docs](./addons/packages/velero) |
-| Multus CNI | Provides ability for attaching multiple network interfaces to pods in Kubernetes | [Multus CNI package docs](./addons/packages/multus-cni) |
 | Whereabouts | Provides A CNI IPAM plugin that assigns IP addresses cluster-wide | [Whereabouts package docs](./addons/packages/whereabouts) |
-| Sriov Network Device Plugin | The SR-IOV Network Device Plugin is Kubernetes device plugin for discovering and advertising SR-IOV virtual functions (VFs) available on a Kubernetes host. | [Sriov Network Device Plugin package docs](./addons/packages/sriov-network-device-plugin) |
 
 ## Contributing
 
 If you are ready to jump in and test, add code, or help with documentation,
-follow the instructions on our [Contribution Guidelines](./CONTRIBUTING.md) to
+follow the instructions on our [Contribution Guidelines](https://tanzucommunityedition.io/docs/latest/contribute/contributing/) to
```

Comment:
Shouldn't this be a relative link?

Reply:
If I want the contributing file to open in the docs site - I thought no?
Can I do a relative doc link when the readme is outside the Hugo root `/site`.

Reply:
Ah, sorry. You are right with the way you have this. I misunderstood what was going on.